### DISCUSSION TITLE
Messaging Insights <> Paste

### DISCUSSION THREAD

**Discussion by dreamealist** [twilio-labs/paste#3234] [Office hours] [answered]
### Background

I'm working on new customer-facing UX features for Messaging Insights & Logs. This area has not been touched for a long time, and the Messaging Insights dashboard under the Monitoring tab has legacy UI components and styles. 
[Current Messaging Insights UIs](https://www.figma.com/file/Yft7tlkKuURYCrkYGiV8YW/Msg-%3C%3E-Voice-Insights-and-Intelligent?type=whiteboard&node-id=0%3A1&t=67I6nBLlHRtaGSgV-1)

### Feedback

1) I want to share the recent updates in Messaging Insights design and kick off this conversation about this product area.
2) I also wonder how the new Paste design system will affect this part of the Console since some parts of the UIs have legacy styles and components, as it was built four years ago and hasn't been updated much.
3) For these new Insights features, I designed new data visualization & charts for new insights features. I want to sync with the Paste team on how we can collaborate on defining data viz in Paste.
4) I'm also working on Intelligent Alerts features within Messaging Insights, which we're targeting to go GA in Q3. For this experience, I'm using the new Paste components and want to get Paste input if this is the right approach.

### Relevant links

1. Messaging Insights - [Scheduled messages](https://www.figma.com/file/ihZXXI8viKlyRDQbkM9otu/Insights---Link-Shortening?type=design&node-id=694%3A212002&t=HTpQ7KgeVsSn3wPb-1)
2. Messaging Insights - [Link shortening](https://www.figma.com/file/ihZXXI8viKlyRDQbkM9otu/Insights---Link-Shortening?type=design&node-id=694%3A212002&t=HTpQ7KgeVsSn3wPb-1)
3. [Intelligent Alerts](https://www.figma.com/file/O0kqLRyP8FJUXka76XBCz4/Intelligent-Alerts-%26-Recommendation?type=design&node-id=369%3A375930&t=g9OxyyAMjZDzW8Rr-1)

### Are you looking for visual design support?

- [ ] Yes

**Comment by loreina** [comment, marked as answer]
Hi @dreamealist 

Thanks for coming to office hours! You can find the [office hours recording here](https://www.loom.com/share/ccd535a15f08469b8920ca12fd7af286) and the password is available in the #help-design-system channel in Slack.
